Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,30 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen = send ? ends->numsendopen : ends->numrecvopen;
```

Comment:
Ideally I'd assert that the (mandatory) pointer parameters are non-null rather than assuming it's the case.